src/dp_abstraction/aca_dataplane_ovs.cpp

```diff
@@ -30,6 +30,9 @@ using namespace aca_ovs_l2_programmer;
 using namespace aca_ovs_l3_programmer;
 using namespace aca_zeta_programming;
 
+extern std::atomic<int> packet_in_counter;
```

Comment:
Do we want to keep these 2 counters? In master ACA

Reply:
We shouldn't keep them, they are removed.